osu.Game/Screens/Edit/Screens/Setup/Components/LabelledComponents/LabelledSwitchButton.cs

```diff
@@ -0,0 +1,138 @@
+﻿// Copyright (c) 2007-2018 ppy Pty Ltd <contact@ppy.sh>.
+// Licensed under the MIT Licence - https://raw.githubusercontent.com/ppy/osu/master/LICENCE
+
+using OpenTK;
+using OpenTK.Graphics;
+using osu.Framework.Allocation;
+using osu.Framework.Configuration;
+using osu.Framework.Graphics;
+using osu.Framework.Graphics.Containers;
+using osu.Framework.Graphics.Shapes;
+using osu.Framework.Graphics.UserInterface;
+using osu.Game.Graphics;
+using osu.Game.Graphics.Sprites;
+
+namespace osu.Game.Screens.Edit.Screens.Setup.Components.LabelledComponents
+{
+    public class LabelledSwitchButton : CompositeDrawable, IHasCurrentValue<bool>
+    {
+        private readonly Box background;
+        private readonly OsuSpriteText label;
+        private readonly OsuSpriteText bottomText;
+
+        private const float corner_radius = 15;
+        private const float default_label_text_size = 16;
+        private const float default_bottom_label_text_size = 12;
+        private const float normal_height = 40;
```

Comment:
why specify "normal" if the "extended" height is not in a variable but inline? make it a variable too.